### PR TITLE
Add like button for each comment and onclick test function.

### PR DESCRIPTION
**Pull request by kpancha**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -67,16 +67,22 @@ function getAndDisplayComments() {
...
+      commentContainer.innerHTML += '<button class="btn btn-danger" onclick="addLike(\'' + comment.ID + '\');">Like</button>';
+      commentContainer.innerHTML += '<br><br>';
+      document.getElementById('content').appendChild(commentContainer);
     }
   });
 }
+
+function addLike(id) {
+  console.log('func called with param ' + id);
+}
```
Comment (kpancha) [inline portfolio/src/main/webapp/script.js:88]: Functionality needs to be implemented

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -76,13 +76,12 @@ function getAndDisplayComments() {
       commentContainer.innerHTML += comment.CONTENT + '<br>';
       commentContainer.innerHTML += comment.numLikes + ' likes<br>';
       commentContainer.innerHTML += 'Posted at ' + comment.TIMESTAMP + '<br>';
-      commentContainer.innerHTML += '<button class="btn btn-danger" onclick="addLike(\'' + comment.ID + '\');">Like</button>';
+      const likeButtonHTML = '<form action="/data" method="POST">' + 
+        '<input type="hidden" name="comment-id" value="' + comment.ID + '" />' + 
+        '<input type="submit" value="Like" class="btn btn-danger btn-sm" /></form>';
```
Comment (kpancha) [inline portfolio/src/main/webapp/script.js:81]: Hard coding HTML in my js file doesn't seem like the cleanest way to do this but I couldn't get it to work exactly how I wanted when using document.createElement functions. Considering adding a TODO to come back to this later this week?

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 3]
```diff
@@ -9,6 +9,8 @@
   private final String CONTENT;
   private int numLikes;
   private final Date TIMESTAMP;
+  private final int ID;
+  private static int numComments = 0;
```
Comment (jesseelizabeth) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:13]: All of these fields should be camelCase. Caps are used for static final variables:  https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 4]
```diff
@@ -21,6 +23,7 @@ public Comment(String name, String content) {
     CONTENT = content;
     numLikes = 0;
     TIMESTAMP = new Date();
+    ID = numComments++;
```
Comment (jesseelizabeth) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:26]: These should all use this: `this.content = content` etc.

**Comment by jesseelizabeth** [comment]
Looks like all of this functionality is implemented and updated in the following PR. Do we need this one anymore?

**Pull request closed without merging**